Lab4.ipynb

```diff
@@ -0,0 +1,713 @@
mAh0Bc4C5iS3m0KcHb672cBTzrn6p1zK4FlePz+36bFCTPrvd8/z4HsX+ev0eJMyRnPCLeD43TnVSRyUhbj4ZwJ9HPVfC25LOw4IhKUGQ/Gx9GdWs7SnVeRSHowNo5SGjg/tSjsKCKSAWY2CJgIfAz0dM5thL0FDGDfElt9gbX7/bd16a8dUKsVJ8zsCeAjYKSZrTOzq4Gfm9lcM5sDHA/c2lrnz6S3YwNZYWV8s2k2ORovJxI5f4/1Y55159KmueTpzqtI5My1Hkyz3lyYWkix7ryKRM5y68zbNoBvuCV0cbVhxxEJj8viDbrtm/MxvV33ZQ/BzIqBZ4FbnHPNrVP8ZUuANDsUITOLCH/ZWZ276Eu+/GCLjpVKkdy9+yATtVwKeMBV8N98wCkNi3nBhoWWpT3oqGsd5z//cZudK2ODFyorM3Wk0D3gKriTdzirYQFP2qg2OmsEX+s+hr501BptbFsVBc+1TTtWCe2LHnQV3MMbnNcwlwdt7EEfL1Ov47Z8T2uP759t+Zisspr42zOa3acoQ+fy8z4ctYGGqdqDKyo84kZzLGu51M3nN0xq/lwNWVaE7KjDQvWZoKPZ5pyb3NwOZpZgb2Hicefcn9Jf3mxmvZ1zG9MjJbakv74O6L/ff+8HbGju+GGs1hFJn9CLWXTnUhZS4BrDjiMiAc23bnxEby5gMSW68yoSOSusjLcYwDkspavuvIpEzgYr5iWGcBor6euqw44jIgGZmbG3s8FC59wv9/vWC8AV6b9fATy/39cvNLM8MxsMDAemNXcOFSf8MuMBxtKZes5lSdhpRKQFHmIMhTRyIRrzKhJFU6gghuMyFoQdRURa4DHKaSDGN5kfdhSRcDjL3s3b0cBlwAlmNiu9nQb8L3CSmS0FTkr/G+fcfOBpYAHwCnCjc833tVFxIoDF1oV36cd5LKHM1YUdR0QCWmWdeIOBnM0yurmasOOISECbrIgXGcoprKKf7ryKRM5Oy+dZhvMV1jHc7Qw7jogE4Jx73zlnzrlxzrkJ6e0l59x259xXnXPD03/u2O///NQ5N9Q5N9I597LXOVScCOhhKsglxaUsDDuKiLTAFEYDcIXuvIpE0lRGUU+cq7J/wS8R+RJ/ZCS7yOUa5oJrdm48EelgVJwIaL2V8FcG83VWaLycSARtsSKeZygns4rBrv1M+CnSUVRaPk8zgmNZT4XbFnYcEQmoxhI8RjmT2MKhbAo7johkERUnWuAxRtOo8XIikTWVcnaT4Drmhh1FRFrgWUawjXyuY47uvIpE0IsMZT3FXMtcYh11JQzpkMxl75YNPIsTZnaen691JHvHy43gK6xjxGdDakQkInZbLo9TzmQ2M9npro1I1NRZDo9QwWh2cBzrw44jIgE1WYwHGMNgqvgaq8OOIyJZwk/PiZ/4/FqH8kdGsJM83bURiai/MJQNFKXv2qgNi0TN6wxiBZ24mrkkmp/8W0Sy0Pv0ZT5duYL55LumsOOISBY4YHHCzE41s98Cfc3sN/ttjwAd/gpSYwn+wGjGs40j2Rh2HBEJqNHiPMgYhrCLk3TXRiRyUmbcz1j6sIfTWRF2HBEJyoz7GEdX6jiXJWGnEWl9Lsu3LNBcz4kNwHSgDpix3/YC8LXWj5b9XmIwayjhWuYQ13g5kch5j34spAvfZJ7u2ohE0HTrxXR6cikLKHYNYccRkYAWWlfepR/ns5gurjbsOCISsgMWJ5xzs51zU4Bhzrkp+21/ck4LEwMkLcb9jKU/u/m67tqIRE/6rk036vgGS8NOIyIt8HvGUkwjF7Eo7Cgi0gIPMoYcUlyuJb5FOjw/c04cZmavm9kSM1thZivNTL+Jp/2d3sykO5exgELXGHYcEQlovnXjb/TlAhZT5urCjiMiAa20Ml5jEGezjF5uT9hxRCSgjVbMCwzlFFYy0O0KO45IKzJwWbxlAT/FiQeBXwLHAIcCk9N/CvzjzmspDVzMwrDTiEgLPMgYcknqro1IRD1CBSmMK5kXdhQRaYHHKaeWBNdqiW+RDs1PcWKXc+5l59wW59z2fVurJ4uQ5daZNxjIOSyjp+7aiETOeivhLwzlNFYywFWFHUdEAtpuBTzDcE5gLSO1xLdI5FRbHo8zisPZxES3Oew4IhISP8WJt83sdjM70swm7dtaPVnEPJy+a3OV7tqIRNJjlFNHnGt010Ykkp5mpJb4Fomw5xnGJgq5jrmY2rC0V2GvyBHh1Tr2OZy9Qzn+G7gjvf2iNUNF0TYr1F0bkQirsjyeYBRHspHxbkvYcUQkoFpL8CijGaclvkUiqdHiPMQYhlHJV1kTdhwRCYFnccI5d/yXbCe0RbioeZqR7CCPbzFbd21EIujPDGcLBVzHHN21EYkgLfEtEm3v0J/FdOZK5pHrkmHHEZE25lmcMLN//7KtLcJFTa0lmEIFY9jOMWwIO46IBNSQvmszQndtRCIptd8S36driW+RyHFm/J5x9KCWc1kSdhyRzAt76EY7GNaxZ78tCZwKDGrFTJH2CoNYSSnXMJcc3bURiZy3GMBiOnMV88h3TWHHEZGA9i3xfTkLKHYNYccRkYDmWHfepw8XsogurjbsOCLShvwM67hjv+2nwFeAvq2eLKJSFuP3jKMvuzmD5WHHEZGAnBn3MJ7u1HIei8OOIyJBmXEv4ymigUu1PLBIJN3POHJIcSXzw44iIm3IT8+JzysEhmQ6SHsynZ7MoAeX6q6NSCTNt268TT/OZwndXU3YcUQkoBVWxssM5iyW089Vhx1HRALaYMX8ieGczCqGu51hxxHJnLCHbkR9WIeZzTWzOeltPrAY+HXrR4swM+5jHMU06q6NSEQ9wFhiOC0PLBJRU6ignvjepUVFJHKmUs4uTTQv0qH46TlxOnBGejsZ6OOcu6tVU7UDK62Ml9J3bfq7qrDjiEhAW6yIPzKCE1lDudsedhwRCajS8nmcco5kI4e4TWHHEZGAaizBI1Qwjm0cy/qw44hIG8jx2sE5t9rMxgPHp
```

Comment:
I know you probably have this in there to help make sense of the data, but I think once you're all finished with the report you can take these large arrays being printed out.